src/ReactiveDomain.Policy/Application/SecurityPolicySyncService.cs

```diff
@@ -63,10 +64,14 @@ public class SecurityPolicySyncService :
                 var policyId = Policy.PolicyId == Guid.Empty ? Guid.NewGuid() : Policy.PolicyId;
                 var app = new Domain.SecuredApplication(appId, policyId, Policy.ApplicationName, Policy.SecurityModelVersion, Policy.OneRolePerUser, source);
                 _primaryClientId = Guid.NewGuid();
-                
+
                 repo.Save(app);
+
+                var redirectUris = new[] { "http://localhost/elbe", "/root/signin-google" };
+                var logoutRedirectUris = new[] { "http://localhost/elbe" };
+                var frontChannlLogoutUri = "http://localhost/elbe";
```

Comment:
These are temporary, yes? A `TODO` wouldn't be a bad idea here.

Reply:
adopted